### ISSUE TITLE
Feature Request: `android.permission.MANAGE_EXTERNAL_STORAGE` permission

### ISSUE THREAD

**Issue by personalizedrefrigerator** [CypherpunkArmory/UserLAnd#1468]
# Describe the problem your feature request solves.
At present, UserLAnd only requests access to `storage/Media`. In this way, most of my documents (in `/host-fs/sdcard/Documents`) can't be accessed by UserLAnd. Requesting the `all files` permission would fix this.

# Describe the solution you'd like 
Request the [`MANAGE_EXTERNAL_STORAGE`](https://developer.android.com/reference/android/Manifest.permission#MANAGE_EXTERNAL_STORAGE) permission in `AndroidManifest.xml`, but [mark it as optional](https://stackoverflow.com/a/15431005).

# Describe alternatives you've considered
 * Sending files to UserLAnd via KDE Connect/scp
     * I then have duplicated files!

**Comment by Saikatsaha1996** [comment]
yes happened same problem with me 🥲 can access only /host-rootfs/sdcard/Download but shown only folder

**Comment by KingOfPlayer** [comment]
This blocking other stats. if this permission does not exists, can't get Basic thermal sensor info. (Android 11)

**Comment by Saikatsaha1996** [comment]
cp my_file /host-rootfs/sdcard/Download "permission denied" 
mv my_file /host-rootfs/sdcard/Download "permission denied"

Under /host-rootfs/sdcard/Download or (Any Folder) not showing any files only jpeg or jpg can showing..

**Comment by HaraldWeber** [comment]
Writing files to the sdcard is mandatory in my opinion.
Please add this permission!

**Comment by OdinVex** [comment]
Google's economic and sandboxing terrorism continues, this is kind of a priority issue, not a 'feature request'.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
